module/VuFind/src/VuFind/Db/Service/UserService.php

```diff
@@ -82,11 +82,11 @@ public function getUserById(int $id): ?UserEntityInterface
      * Field name must be id, username or cat_id.
      *
      * @param string          $fieldName  Field name
-     * @param int|null|string $fieldValue Field value
+     * @param int|string|null $fieldValue Field value
```

Comment:
This change isn't directly related to the others, but it feels like null should be last...